charts/ecosystem/templates/event-streams-secret.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.eventStreamsSecretName }}
+
+{{- $eventStreamsSecretName := (printf "%s-%s" .Release.Name .Values.eventStreamsSecretName) }}
```

Comment:
If we want to load a value from an existing Secret (like one created from an ExternalSecret), that secret would have to be called `<release-name>-<value>`, otherwise a new secret will always be created.

Maybe we could use the provided `eventStreamsSecretName` value if it's been set and if it hasn't, then we'll generate a name for the new secret?